build/Podmanfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+FROM registry.access.redhat.com/ubi8/ubi
+
+RUN yum install -y java-11-openjdk-devel git
+
+ENV MVN_VERSION=3.8.3 
+
+RUN curl -sS \
+    https://dlcdn.apache.org/maven/maven-3/${MVN_VERSION}/binaries/apache-maven-${MVN_VERSION}-bin.tar.gz \
```

Comment:
link is outdated please update it to https://archive.apache.org/dist/maven/maven-3/${MVN_VERSION}/binaries/apache-maven-${MVN_VERSION}-bin.tar.gz \

Reply:
updated.